roles/logging/tasks/checks.yml

```diff
@@ -1,9 +0,0 @@
----
-- name: install logstash-forwarder process check
-  sensu_check_dict: name="check-logstash-forwarder-process" check="{{ sensu_checks.logging.check_logstash_forwarder_process }}"
-  notify: restart sensu-client missing ok
-  when: logging.forward_type != 'filebeat'
-
-- name: install filebeat process check
-  sensu_check_dict: name="check-filebeat-process" check="{{ sensu_checks.logging.check_filebeat_process }}"
-  when: logging.forward_type == 'filebeat'
```

Comment:
Since you've decided to move this and no longer use `sensu_check_dict`, please remove the following from the `sensu-checks` role.

`sensu_checks.logging.check_logstash_forwarder_process`
`sensu_checks.logging.check_filebeat_process`